cmd/edit/cmd.go

```diff
@@ -21,14 +21,16 @@ import (
 
 	"github.com/openshift/moactl/cmd/edit/cluster"
 	"github.com/openshift/moactl/cmd/edit/ingress"
+	v "github.com/openshift/moactl/cmd/validations"
 	"github.com/openshift/moactl/pkg/interactive"
 )
 
 var Cmd = &cobra.Command{
-	Use:     "edit RESOURCE [flags]",
-	Aliases: []string{"update"},
-	Short:   "Edit a specific resource",
-	Long:    "Edit a specific resource",
+	Use:              "edit RESOURCE [flags]",
+	Aliases:          []string{"update"},
+	Short:            "Edit a specific resource",
+	Long:             "Edit a specific resource",
+	PersistentPreRun: v.Validations,
```

Comment:
This is probably not necessary in here, as it will show up for _all_ resources being editted. Also, this command needs an existing running cluster anyway, and if the cluster exists in OCM, we can be certain that the corresponding CF Stack also exists

Reply:
Addressed and resolved